BLE_SM/README.md

```diff
@@ -0,0 +1,27 @@
+# SM - example usage of the Security Manager to pair and encrypt
+
+Demonstration of possible usage of the Security Manager. Security Manager deals with pairing, authentication and encryption.
+
+The application demonstrates usage as a central and a peripheral. The central will connect to any connectable device present. Please have one ready and advertising. Application will attempt pairing. Please authorise your peer device to pair.
+
+Upon success it will disconnect and start advertising to demonstrate usage as a peripheral. Please scan and connect using your peer device. Upon connection grant pairing if prompted. Upon success the application will disconnect. Observe the terminal to keep track of the sequence. 
```

Comment:
Phase two will start whether phase 1 succeed or not. Could you indicate that if phase 1 fail, the peripheral will request pairing or if phase 1 succeeded then the link is encrypted.

Reply:
reversed the order